modules/assisted-installer-adding-hosts-to-the-cluster.adoc

```diff
@@ -0,0 +1,25 @@
+// This is included in the following assemblies:
+//
+// assisted-installer-installing.adoc
+
+:_content-type: PROCEDURE
+[id="adding-hosts-to-the-cluster_{context}"]
+= Adding hosts to the cluster
+
+After setting the cluster details and optionally configuring static networking, you must add one or more hosts to the cluster. Adding a host to the cluster involves generating a discovery ISO. The discovery ISO runs {op-system-first} in-memory with an agent. For each host, use the following procedure.
+
+.Procedure
+
+. Click the *Add hosts* button and select the installation media.
```

Comment:
Actually, it's a wizard. I took out the "After..." sentence, because the sequence in the user-interface keeps changing. These docs will actually move out of the OCP docs shortly anyway, because Assisted Installer docs change much more frequently than OCP docs.